### ISSUE TITLE
~/.osfv/snipeit.yml is not created automatically as stated in documentation

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/osfv-scripts#17]
According to https://github.com/Dasharo/osfv-scripts/blob/main/osfv_cli/README.md#usage ~/.osfv/snipeit.yml should be created automatically, but it didn't happen in my case.

**Comment by macpijan** [comment]
Hmm, do you have some log from `make install`?

```
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ rm -rf ~/.osfv
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ ls ~/.osfv    
ls: cannot access '/home/macpijan/.osfv': No such file or directory
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ make install
poetry build
Building osfv_cli (0.3.1)
  - Building sdist
  - Built osfv_cli-0.3.1.tar.gz
  - Building wheel
  - Built osfv_cli-0.3.1-py3-none-any.whl
pip install dist/osfv_cli-0.3.1-py3-none-any.whl
Defaulting to user installation because normal site-packages is not writeable
Processing ./dist/osfv_cli-0.3.1-py3-none-any.whl
Requirement already satisfied: PyYAML<7.0,>=6.0 in /home/macpijan/.local/lib/python3.11/site-packages (from osfv-cli==0.3.1) (6.0.1)
Requirement already satisfied: importlib-resources<7.0.0,>=6.1.1 in /home/macpijan/.local/lib/python3.11/site-packages (from osfv-cli==0.3.1) (6.1.1)
Requirement already satisfied: paramiko<4.0.0,>=3.2.0 in /home/macpijan/.local/lib/python3.11/site-packages (from osfv-cli==0.3.1) (3.2.0)
Requirement already satisfied: pexpect<5.0.0,>=4.8.0 in /usr/lib/python3.11/site-packages (from osfv-cli==0.3.1) (4.8.0)
Requirement already satisfied: requests<3.0.0,>=2.31.0 in /home/macpijan/.local/lib/python3.11/site-packages (from osfv-cli==0.3.1) (2.31.0)
Requirement already satisfied: unidecode<2.0.0,>=1.3.6 in /home/macpijan/.local/lib/python3.11/site-packages (from osfv-cli==0.3.1) (1.3.6)
Requirement already satisfied: bcrypt>=3.2 in /home/macpijan/.local/lib/python3.11/site-packages (from paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (4.0.1)
Requirement already satisfied: cryptography>=3.3 in /usr/lib64/python3.11/site-packages (from paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (37.0.2)
Requirement already satisfied: pynacl>=1.5 in /home/macpijan/.local/lib/python3.11/site-packages (from paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (1.5.0)
Requirement already satisfied: ptyprocess>=0.5 in /usr/lib/python3.11/site-packages (from pexpect<5.0.0,>=4.8.0->osfv-cli==0.3.1) (0.7.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/lib/python3.11/site-packages (from requests<3.0.0,>=2.31.0->osfv-cli==0.3.1) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3.11/site-packages (from requests<3.0.0,>=2.31.0->osfv-cli==0.3.1) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/lib/python3.11/site-packages (from requests<3.0.0,>=2.31.0->osfv-cli==0.3.1) (1.26.18)
Requirement already satisfied: certifi>=2017.4.17 in /home/macpijan/.local/lib/python3.11/site-packages (from requests<3.0.0,>=2.31.0->osfv-cli==0.3.1) (2023.5.7)
Requirement already satisfied: cffi>=1.12 in /usr/lib64/python3.11/site-packages (from cryptography>=3.3->paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (1.15.1)
Requirement already satisfied: pycparser in /usr/lib/python3.11/site-packages (from cffi>=1.12->cryptography>=3.3->paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (2.20)
Requirement already satisfied: ply==3.11 in /usr/lib/python3.11/site-packages (from pycparser->cffi>=1.12->cryptography>=3.3->paramiko<4.0.0,>=3.2.0->osfv-cli==0.3.1) (3.11)
osfv-cli is already installed with the same version as the provided wheel. Use --force-reinstall to force an installation of the wheel.

[notice] A new release of pip is available: 23.2.1 -> 24.0
[notice] To update, run: pip install --upgrade pip
test -f /home/macpijan/.osfv/snipeit.yml || install -D -m 644 snipeit.yml /home/macpijan/.osfv/snipeit.yml
test -f /home/macpijan/.osfv/zabbix.yml || install -D -m 644 zabbix.yml /home/macpijan/.osfv/zabbix.yml
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ ls ~/.osfv  
snipeit.yml  zabbix.yml
```

**Comment by pietrushnic** [comment]
My bad, for some reason I translatd `~/.osfv` to `~/.config/osfv` in my head. Sorry for the noise.
